demos/interactive_face_detection_demo/cpp_gapi/main.cpp

```diff
@@ -456,10 +456,10 @@ int main(int argc, char *argv[]) {
                 throw std::invalid_argument(msg.str());
             }
 
-            timer.start("total");
+            auto startTime = std::chrono::steady_clock::now();
```

Comment:
const auto

Does `PerformanceMetrics::update(...)` function change `startTime`?

Reply:
No, it doesn't. 
Previously I placed start time measurement in the wrong place, so latency was measured incorrectly (increasing with each frame). I fixed it, review please.

Reply:
> Previously I placed start time measurement in the wrong place, so latency was measured incorrectly (increasing with each frame). I fixed it, review please.

Not relevant. Removed latency output in gapi demos.

Reply:
Ok.